pre-docs/bolt_command_reference.md

```diff
@@ -1,109 +1,498 @@
+| `--description DESCRIPTION` | Description to use for the job. |
+| `--params PARAMETERS` | Parameters to a task or plan as json, a json file `@<file>`, or on stdin `-`. |
+| **Authentication** |
+| `-u`, `--user USER` | User to authenticate as. |
+| `-p`, `--password [PASSWORD]` | Password to authenticate with. <br> Omit the value to prompt for the password. |
+| `--private-key KEY` | Private SSH key to authenticate with. |
+| `--[no-]host-key-check` | Check host keys with SSH. |
+| `--[no-]ssl` | Use SSL with WinRM. |
+| `--[no-]ssl-verify` | Verify remote host SSL certificate with WinRM. |
+| **Escalation** |
+| `--run-as USER` | User to run as using privilege escalation. |
+| `--sudo-password [PASSWORD]` | Password for privilege escalation. <br> Omit the value to prompt for the password. |
+| **Run Context** |
+| `-c`, `--concurrency CONCURRENCY` | Maximum number of simultaneous connections. | 100 |
+| `--compile-concurrency CONCURRENCY` | Maximum number of simultaneous manifest block compiles. | Number of cores |
+| `-m`, `--modulepath FILEPATHS` | List of directories containing modules, separated by `:`. <br> Directories are case-sensitive. |
+| `--boltdir FILEPATH` | Specify what Boltdir to load config from. | Autodiscovered from current working directory. |
+| `--configfile FILEPATH` | Specify where to load config from. | `~/.puppetlabs/bolt/bolt.yaml` |
+| `-i`, `--inventoryfile FILEPATH` | Specify where to load inventory from. | `~/.puppetlabs/bolt/inventory.yaml` |
+| `--[no-]save-rerun` | Whether to update the rerun file after this command. |
+| **Transports** |
+| `--transport TRANSPORT` | Specify a default transport. <br> `ssh`, `winrm`, `pcp`, `local`, `docker`, `remote` |
+| `--connect-timeout TIMEOUT` | Connection timeout. | Varies |
+| `--[no-]tty` | Request a pseudo TTY on nodes that support it. |
+| `--tmpdir DIR` | The directory to upload and execute temporary files on the target. |
+| **Display** |
+| `--format FORMAT` | Output format to use. <br> `human`, `json` |
+| `--[no-]color` | Whether to show output in color. |
+| `-v`, `--[no-]verbose` | Display verbose logging. |
+| `--trace` | Display error trace stacks. |
+
+
+## `plan show`
+
+Show a list of available plans or details for a specific plan.
+
+### Usage
+
+`bolt plan show [PLAN]`
+
+- Specify an available plan to show documentation for the plan.
+
+### Options
+
+| Option | Description | Default |
+|--------|-------------|---------|
+| **Run Context** |
+| `-m`, `--modulepath FILEPATHS` | List of directories containing modules, separated by `:`. <br> Directories are case-sensitive. |
+| `--boltdir FILEPATH` | Specify what Boltdir to load config from. | Autodiscovered from current working directory. |
+| `--configfile FILEPATH` | Specify where to load config from. | `~/.puppetlabs/bolt/bolt.yaml` |
+
+
+## `puppetfile install`
+
+Install modules from a Puppetfile into a Boltdir.
```

Comment:
A few lines below, `Boltdir` has backticks. Are they deliberately left off in this line?

Come to think of it, there are lots of references to Boltdir in this file. I wonder if it makes the most sense to use backticks when referring to an actual directory named `Boltdir`, and no backticks when referring to the concept of a Boltdir. Maybe like this?

"Move the file to your Boltdir"
vs.
"Move the file to `Boltdir`"

If you're cool with that sort of change, we should do a search & replace to find all references in this file.

Reply:
Good catch. I removed backticks from all instances since it's always referring to the concept of a Boltdir.